core/math/math_funcs.h

```diff
@@ -34,14 +34,15 @@
 #include "core/math/math_defs.h"
 #include "core/typedefs.h"
 
+#include "random_base.h"
```

Comment:
Should be `core/math/random_base.h` (and moved two lines up after `math_defs.h`).